Dockerfiles/ci/run_tests.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+
+import unittest
+import os
+import os.path
+from os import path
+
+class RunOperatorTestPlaybookTests(unittest.TestCase):
+
+    def setUp(self):
+        self.OPERATOR_DIR = os.getenv('OPERATOR_DIR',
+                                      "/project/operator_dir/")
+        self.OPERATOR_WORK_DIR = os.getenv('OPERATOR_WORK_DIR',
+                                           "/project/test_operator_work_dir/")
+        self.WORK_DIR = os.getenv('WORK_DIR',
+                                  "/project/output/")
+        self.PLAYBOOKS_DIR = os.getenv('PLAYBOOKS_DIR',
+                                       "/project/operator-test-playbooks/")
+
+    def test_for_report_failed_empty_alm_examples(self):
+        EXEC_CMD = "ansible-playbook -vvv -i localhost, --connection local \
+                    operator-test-playbooks/prepare-operator-metadata.yml \
+                    -e 'operator_dir={operator_dir}' \
+                    -e 'operator_work_dir={operator_work_dir}' \
+                    -e'work_dir={work_dir}'".format(operator_dir=self.OPERATOR_DIR,
+                                                    operator_work_dir=self.OPERATOR_WORK_DIR,
+                                                    work_dir=self.WORK_DIR)
+        os.system(EXEC_CMD)
+        self.assertEqual(True,
+                         path.exists("{}/parsed_operator_data.yml".format(self.WORK_DIR)))
+        self.assertEqual(True,
+                         path.exists("{}/parse_operator_metadata_results.json".format(self.PLAYBOOKS_DIR)))
```

Comment:
There is even `assertTrue`; you needn't use `self.assertEqual(True,`

https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertTrue

Reply:
updated